### DISCUSSION TITLE
Using flexget to automate generic scripted downloads

### DISCUSSION THREAD

**Discussion by navels** [Flexget/Flexget#3278] [Q&A] [answered]
I have a script, using curl, for logging into and downloading videos from a site that I am subscribed to.  They don't have an RSS feed or, as far as I can tell, any other standard publishing mechanisms that would fit into a built-in flexget use case or plugin.

Is there some way to use flexget to schedule my script and to detect new videos and download them?  I can tailor the output of the script to whatever is easiest to get this to work.

**Comment by paranoidi** [comment, marked as answer]
Best solution would be to write it in python and as a custom user plugin. If that is not possible you could always write a CSV file for example and we have an input for that. FlexGet will not be able to access any links that require login though unless you pass cookies too ...

**Reply by navels** [reply]
Cool, thanks, I will take a crack at it.  And yep, am using cookies :-)
